### PR TITLE
Correct copyright information

### PR DESCRIPTION
**Pull request by Simon-1-1**

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -1,5 +1,3 @@
-Copyright 2022
-
```
Comment (plaos) [inline NOTICE:2]: Why should this be removed?

Comment (Simon-1-1) [reply NOTICE:2]: I think it's because Volvo isn't mentioned in the context so it's an incomplete statement. Was asked by The Council to remove it.

**Comment by Simon-1-1** [comment]
Updated header comments in Python files as well.

**Review by jzoran — APPROVED** [review]

File: NOTICE (hunk 1 above)
Comment (plaos) [reply NOTICE:2]: Makes sense!

**Review by plaos — APPROVED** [review]
LGTM

**Pull request merged**